README.md

```diff
@@ -43,20 +43,24 @@ More information about getting started can be found on the team confluence.
 1. Open a terminal window
 1. `git clone https://github.com/SFDigitalServices/sf-dahlia-web.git` to create the project directory
    - Using gh is recommended. This can be installed with either [Brew](https://brew.sh/) or downloading directly from [Github](https://cli.github.com/)
-3. `cd sf-dahlia-web` to open the directory
-4. Using NVM, install 18.12.1 (or whatever version we are on) with `nvm install 18.12.1`
-5. Using RVM, install 3.1.3 (or whatever version we are on) with `rvm instal 3.1.3`
-6. `bundle install` to download all necessary gems
+1. `cd sf-dahlia-web` to open the directory
+1. Using NVM, install 18.12.1 (or whatever version we are on) with `nvm install 18.12.1`
+1. Using RVM, install 3.1.3 (or whatever version we are on) with `rvm instal 3.1.3`
+1. `bundle install` to download all necessary gems
    - See [here](https://stackoverflow.com/a/19850273/260495) if you have issues installing `pg` gem with Postgres.app, you may need to use: `gem install pg -v <failing-pg-version> -- --with-pg-config=/Applications/Postgres.app/Contents/Versions/latest/bin/pg_config`
    - If you need to run this command make sure you run bundle install again following the success of the Postgres installation to install the remaining gems
-7. `yarn install` to install bower, grunt and other dependencies (which will also automatically `bower install` to load front-end JS libraries)
-8. `overcommit --install` to install git hooks into the repo
-9. Download PostgreSQL. You only need to turn it on, step 10 will set it up for you.
-10. `rake db:create && rake db:migrate` to create the dev database and migrate the DB tables
-11. copy `.env.sample` into a file called `.env`, and copy correct Salesforce environment credentials (not shared publicly in this repo)
-12. `./bin/webpack-dev-server` to start the webpack dev server
+1. `yarn install` to install bower, grunt and other dependencies (which will also automatically `bower install` to load front-end JS libraries)
+1. `overcommit --install` to install git hooks into the repo
+1. Download PostgreSQL. You only need to turn it on, the next step will set it up for you.
+1. `rake db:create && rake db:migrate` to create the dev database and migrate the DB tables
+1. copy `.env.sample` into a file called `.env`, and copy correct Salesforce environment credentials (not shared publicly in this repo)
+1. `./bin/webpack-dev-server` to start the webpack dev server
```

Comment:
This command on it's own has never worked for me. I've had to use `NODE_OPTIONS=--openssl-legacy-provider ./bin/webpack-dev-server`, which also happens to the command that `yarn client` aliases

Has anyone else run into this?